### DISCUSSION TITLE
vercel link

### DISCUSSION THREAD

**Discussion by Maha1431** [vercel/community#6495] [Help]
### Summary

after deployed project at vercel ony vercel link not working  its showig blank

### Steps to Reproduce

https://bot-ai-git-main-mahas-projects-3e3aea2d.vercel.app/

**Comment by versecafe** [comment]
@Maha1431 When you check the deployment summary on vercel what does the output look like?

**Comment by Maha1431** [comment]
Afternoon, vercel output is blank

**Reply by versecafe** [reply]
Did you run `vercel --prod` after linking to make sure it runs a deployment?

**Comment by Maha1431** [comment]
Yes, I did.
